srcpkgs/srb2/template

```diff
@@ -0,0 +1,41 @@
+# Template file for 'srb2'
+pkgname=srb2
+version=2.2.8
+revision=1
+wrksrc="SRB2-SRB2_release_${version}"
+create_wrksrc=true
+build_wrksrc=${wrksrc}
```

Comment:
This can be a bit funky, please spell it out fully `$pkgname-$version`

Reply:
Fixed.